exporters/jaeger/src/main/java/io/opentelemetry/exporters/jaeger/Adapter.java

```diff
@@ -95,7 +95,8 @@ private Adapter() {}
     target.addTags(
         Model.KeyValue.newBuilder()
             .setKey(KEY_SPAN_STATUS_MESSAGE)
-            .setVStr(span.getStatus().isOk() ? "" : span.getStatus().getDescription())
+            .setVStr(
```

Comment:
Removed the check for `isOk`, as it is possible user may want to include a description even with an Ok status. However if not included by convention, I can update to just `isOk() || getDescription() == null ? ""....`.

Reply:
I think that's fine.